include/code_800D090.h

```diff
@@ -1,6 +1,6 @@
 #ifndef GUARD_CODE_800D090_H
 #define GUARD_CODE_800D090_H
 
-void sprintf_2(char *buffer, const char *text, ...);
+void sprintfStatic(char *buffer, const char *text, ...);
```

Comment:
Why Static?

Reply:
This is what pmdsky-debug calls it. They have this to say about it:
```
        Functionally the same as Sprintf, just defined statically in many different places.
        
        Since this is essentially just a wrapper around vsprintf(3), this function was probably statically defined in a header somewhere and included in a bunch of different places. See the actual Sprintf for the one in libc.
```